### DISCUSSION TITLE
Account creation problem

### DISCUSSION THREAD

**Discussion by Mojolaoluwa-G** [vercel/vercel#10000] [Help] [answered]
### Problem Description

I'm a New user and I'm having problems getting my account. I'm trying to login after linking vercel with my GitHub account and it's telling me my account is blocked. What do I do? This is not a good user experience at all.

**Comment by TooTallNate** [comment, marked as answer]
You should contact support@vercel.com and they can help get you sorted out. Sorry about the troubles!

**Comment by alvinsjoy** [comment]
Hi I am having the same problem
![image](https://github.com/vercel/vercel/assets/89687635/2c82a63e-0315-4130-9db0-c68a5e31e24c)
![image](https://github.com/vercel/vercel/assets/89687635/3da6e3e2-2d37-405f-9f6a-ab13252dbe11)

**Reply by qingyin741** [reply]
the same problem and don't know how to solve it

**Reply by alvinsjoy** [reply]
Well you can mail about it to support@vercel.com and they'll unblock your account for you.
![Screenshot_2023-07-22-12-15-16-06_e307a3f9df9f380ebaf106e1dc980bb6.jpg](https://github.com/vercel/vercel/assets/89687635/7c30257b-ec01-4098-a748-a5ff9df80196)

**Reply by qingyin741** [reply]
After I received this email, there was no news
